css/templates/travel/page-vars.css

```diff
@@ -112,3 +137,95 @@
   @custom-media --breakpoint-md-lg (min-width: 52.06rem) and (max-width: 72rem);
   @custom-media --breakpoint-v-md-lg (min-height: 52.06rem) and (max-height: 72rem);
 }
+
+body {
+  color: var(--font-color-primary);
+  font-family: var(--font-family-primary);
+  line-height: var(--line-height-2);
+  box-sizing: border-box;
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+}
+
+*, *::before, *::after {
+  box-sizing: inherit;
+}
+
+h1, h2, h3, h4, h5, h6, p {
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+/* Util classes. */
+.min-0 {
+  min-width: 0;
+}
+
+.layout-fill {
+  width: 100%;
+}
+
+.pointer {
+  cursor: pointer;
+}
+
+.focus:focus {
+  outline: none;
+}
+
+*:focus .focus-child {
```

Comment:
This is really bad for a11y, are you sure we want to do this

Also doing this on * is not a good idea.